spec/definitions/CheckoutPage.yaml

```diff
@@ -3,10 +3,10 @@ required:
   - name
   - planId
   - websiteId
-  - urlPathSegment
 properties:
   id:
-    description: Checkout page identifier string
+    description: Your own custom url path segment for this checkout page. It will be appended to checkout url https://checkout.rebilly.com/checkout/`id`.
```

Comment:
> It will be appended to checkout url `https://checkout.rebilly.com/checkout/{id}/`

I guess URL-template is familiar for many peeps.

Reply:
> I guess URL-template is familiar for many peeps.

From what sources?

Reply:
What do you mean?